### PR TITLE
Add dummy source and sink

### LINKED ISSUE #46: Add dummy source and sink to Docker environment.
May be helpful:
* `pl.touk.nussknacker.engine.flink.util.sink.EmptySink`
* `pl.touk.nussknacker.engine.flink.util.transformer.PeriodicSourceFactory`

### PR DESCRIPTION
**Pull request by kantoniak**
* Register source and sink in config
* Copy required `prinz-sample`'s dependencies

Closes #46.

### CODE REVIEW

File: dev-environment/create_environment.sh [hunk 1]
```diff
@@ -1,12 +1,18 @@
 #!/bin/sh
 
+scalaV="2.12"
+nussknackerV="0.2.2"
+
 # Compile and package prinz if not packaged yet
 cd .. &&
-./sbtwrapper prinz/package prinz_sample/package &&
+./sbtwrapper prinz/package prinz_sample/package prinz_sample/pack &&
```
Comment (avan1235) [inline dev-environment/create_environment.sh]: do we need the package command when pack is used?

Comment (avan1235) [reply dev-environment/create_environment.sh]: I think that pack depends on package so it's unused here for prinz_simple

**Review by avan1235 — CHANGES_REQUESTED** [review]

File: dev-environment/create_environment.sh (hunk 1 above)
Comment (kantoniak) [reply dev-environment/create_environment.sh]: Yes, it's for exporting dependency jars. Built-in `package` can't do anything like that.

Comment (avan1235) [reply dev-environment/create_environment.sh]: I mean that package command may be unused  here (as pack may depands on it and call it in background). Let's check it but commit is approved :wink:

**Review by avan1235 — APPROVED** [review]

File: dev-environment/create_environment.sh (hunk 1 above)
Comment (kantoniak) [reply dev-environment/create_environment.sh]: `pack` depends on `package` and I could remove call to `prinz_sample/package` eventually.

**Comment by kantoniak** [comment]
I moved dependencies to an array after approval, [see here](https://github.com/prinz-nussknacker/prinz/pull/60/files#diff-ba8b0ded3aaa0356b50af57388ef877e7177b61451cb98b5eec66747c0f56577R6).

**Pull request merged**